### PR TITLE
Add XIP-47: Group Chat Permissions

### PR DESCRIPTION
**Pull request by cameronvoell**

### CODE REVIEW

File: XIPs/xip-47-group-chat-permissions.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+## Backward compatibility
+
+In addition to adding the new **Permission Policies** and **Permission Options** and making **Permissions Updatable**, we will also make the permission system itself updateable in the following ways:
+
+- `libxmtp` can be updated so existing groups can add new **Permission Policies** via the proto `map<string, MetadataPolicy> update_metadata_policy`. This means that if later on we want to add a new Permission Policy to groups, such as "The Ability to Mute Group Members," we can add it without breaking any existing groups.
+
+- Deprecation and upgrade of the Permissions or Metadata Extensions in an existing group's Group Context
+
+- Existing groups will not be able to include an entirely new Permissions or Metadata Extension until all group members update to a new version of libxmtp and perform a "leaf node update" commit they updated their "supported capabilities."
```
Comment (jhaaaa) [inline XIPs/xip-47-group-chat-permissions.md:109]: @cameronvoell I think there is a missing or extra word in the line above - I can't tell which. =) Might you be able to help, please? Thank you!

Comment (cameronvoell) [reply XIPs/xip-47-group-chat-permissions.md:109]:
```suggestion
- Existing groups will not be able to include an entirely new Permissions or Metadata Extension until all group members update to a new version of libxmtp and perform a "leaf node update commit" that updates their "supported capabilities."  
```

File: XIPs/xip-47-group-chat-permissions.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+The ability to update permissions is needed to address the use cases of initial group permission misconfiguration and the inevitable evolution of a group’s trust dynamic. For example, just because an online project group chat starts as a small group of well-intentioned contributors does not mean that the group may not evolve to have a larger variety of member trustworthiness and contributor types over time.
+
+## Backward compatibility
+
+In addition to adding the new **Permission Policies** and **Permission Options** and making **Permissions Updatable**, we will also make the permission system itself updateable in the following ways:
+
+- `libxmtp` can be updated so existing groups can add new **Permission Policies** via the proto `map<string, MetadataPolicy> update_metadata_policy`. This means that if later on we want to add a new Permission Policy to groups, such as "The Ability to Mute Group Members," we can add it without breaking any existing groups.
+
+- Deprecation and upgrade of the Permissions or Metadata Extensions in an existing group's Group Context
```
Comment (jhaaaa) [inline XIPs/xip-47-group-chat-permissions.md:107]:
```suggestion
- Deprecation and update of the Permissions or Metadata Extensions in an existing group's Group Context
```

[the remark this answers is not in the document]
Comment (jhaaaa) [reply XIPs/xip-47-group-chat-permissions.md]: Addressed!

**Review by jhaaaa — APPROVED** [review]
Thank you for the clear, concise, and important XIP, @cameronvoell and @eleanorhofstedt! So happy to have this ready for community feedback.

File: XIPs/xip-47-group-chat-permissions.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
+---
+xip: 47
+title: Group Chat Permissions
+description: This proposal describes a simple, flexible, updatable group chat permissions system.
+author: Cameron Voell (@cameronvoell), Eleanor Hofstedt (@eleanorhofstedt)
+discussions-to: TBD
```
Comment (jhaaaa) [inline XIPs/xip-47-group-chat-permissions.md:6]:
```suggestion
discussions-to: https://community.xmtp.org/t/xip-47-group-chat-permissions/651
```

**Pull request merged**